src/GoogleMap.jsx

```diff
@@ -18,7 +18,12 @@ const Map = withGoogleMap((props) => {
     radius,
     circleOptions
   } = props;
-
+  let circle = (radius == -1) ?
```

Comment:
This condition is wrong :smile_cat: .
When a user supply -1 as radius value, Circle should be hidden right?
`const circle = (radius !== -1)` : (
  // Show circle
): null